src/Raven.Client/ServerWide/Operations/SetDatabaseDynamicDistribution.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations
+{
+    public class SetDatabaseDynamicDistribution : IServerOperation<bool>
```

Comment:
Do we want to expose this as a part of public API? Maybe internal is enough?

Reply:
why not? I think it was just missing and it is not breaking anything

Reply:
ok